post_assets.py

```diff
@@ -0,0 +1,75 @@
+import sys
+import os
+import csv
+import django
+
+project_dir = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(project_dir)
+django.setup()
```

Comment:
for this to work we might need to specify `DJANGO_SETTINGS_MODULE` eg `os.environ.setdefault("DJANGO_SETTINGS_MODULE", "settings")` as it is in `manage.py`